airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/migrators/BigQueryDestinationState.kt

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.migrators
+
+import io.airbyte.integrations.base.destination.typing_deduping.migrators.MinimumDestinationState
+
+data class BigQueryDestinationState(private val needsSoftReset: Boolean) : MinimumDestinationState {
```

Comment:
you could also just use MinimumDestinationState.Impl instead of creating a new class... though we eventually want destination state, so :shrug:

Reply:
That was the idea, we need it anyway so created one.